lib/phoenix/html/link.ex

```diff
@@ -46,4 +46,40 @@ defmodule Phoenix.HTML.Link do
       end
     end
   end
+
+  @doc """
+  Generates a button that uses a regular HTML form to submit to the given URL.
+  Useful to ensure that links that change data are not triggered by
```

Comment:
Enter a new line here please so they are considered different paragraphs. ExDoc uses the first paragraph as a summary. :)